app/src/main/java/com/example/concordia_campus_guide/Activities/PathsActivity.java

```diff
@@ -131,19 +148,55 @@ public void checkFromToType(Place from, Place to) {
     }
 
     public void setIndoorPaths() {
-        if(!fromIsIndoor && !toIsIndoor) return;
-
+        if (!fromIsIndoor && !toIsIndoor) return;
         //Outdoor -> Indoor
         if (!fromIsIndoor) {
             locationFragment.setIndoorPaths(mViewModel.getEntrance(to), to);
-        }
-        else if(mViewModel.arePlacesSeparatedByATunnel(from, to) || mViewModel.areInSameBuilding(from, to)){
+        } else if (mViewModel.arePlacesSeparatedByATunnel(from, to) || mViewModel.areInSameBuilding(from, to)) {
             locationFragment.setIndoorPaths(from, to);
         }
         //[from -> from_entrance ] + outdoor directions + [to_entrance -> to]
-        else if (toIsIndoor && fromIsIndoor) {
+        else if (toIsIndoor) {
```

Comment:
Can you try getting the directions from an outside location (e.g Dome) to an inside location (e.g VL-some room)? The app keeps crashing for me when I do that and I think it might have to do with this line.

Reply:
Ok I'll try it... The sonarLint told me that it's redundant :P

Reply:
haah yup makes sense.